tools/hyphy/compile.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Remove extra space in front of Emscripten "-s" variables to avoid errors
+emcmake cmake -DCMAKE_EXE_LINKER_FLAGS="${EM_FLAGS//-s /-s} --preload-file data@/hyphy" .
```

Comment:
Did you mean to preload folder `tests/data`? I don't see a `data/` folder in the root path
```suggestion
emcmake cmake -DCMAKE_EXE_LINKER_FLAGS="${EM_FLAGS//-s /-s} --preload-file tests/data@/hyphy" .
```

Reply:
Remnant of copying tn93 over. I've removed the statement.